README.md

```diff
@@ -29,5 +29,118 @@ Ruby on Rails Getting Started.
 `rails server`
 
 ----------------------------
-*SOURCE: *
+**SOURCE:**
 [Development Installation](https://guides.rubyonrails.org/development_dependencies_install.html)
+
+# Deploy to Heroku
+
+This steps are a summary to deploy Rails app to Heroku.
+
+### Upgrade Rails app
+If we have a previous Rails app created we need to add the respective gem, because Heroku does not support `sqlite3`
+
+In `Gemfile` add:
+```
+gem `pg`
+```
+and install the new gem inside the project with:
+```
+bundle install
+```
+Change database adaptor and databases names in `config/database.yml`
+```ruby
+  # ...
+  adapter: postgresql
+  # ...
+  development:
+    <<: *default
+    database: database_development
+  # ...
+  test:
+    <<: *default
+    database: database_test
+  # ...
+  production:
+    <<: *default
+    database: database_production
+    username: user
+    password: password
+  #...(Shorcut to skip lines)
+```
+Finally run in terminal:
+```
+rails db:create
+rails db:migrate
+```
+**Enviroment Configuration Ruby**
+Add next code to `config/environments/production.rb`
+
+```ruby
+config.public_file_server.enabled = ENV['RAILS_SERVE_STATIC_FILES'].present?
+
+if ENV["RAILS_LOG_TO_STDOUT"].present?
+  logger           = ActiveSupport::Logger.new(STDOUT)
+  logger.formatter = config.log_formatter
+  config.logger = ActiveSupport::TaggedLogging.new(logger)
+end
+```
+### Deploy App Heroku
+To made possible this step we should start a git repository. And we need to have installed [heroku-cli](https://devcenter.heroku.com/articles/heroku-cli#download-and-install).
+
+In the project directory type:
+```
+heroku create
+```
+Then we need to push local repository to Heroku:
+```
+git push heroku main
+```
+
+**Create and Migrate Database on Heroku App**
+This line creates a database with the hooby-dev plan:
+```
+heroku addons:create heroku-postgresql:hobby-dev
+```
+Finally we need to migrate database on heroku app:
+```
+heroku run rake db:migrate
+```
+
+### Create Heroku Procfile
+In root path of project we need to add a `Procfile` and write:
+```
+web: bundle exec puma -t 5:5 -p ${PORT:-3000} -e ${RACK_ENV:-development}
```

Comment:
The command you use in the app is a little bit different than this one.

Reply:
I forgot that this is from the documentation. I change it to the original code.